zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/publishing/PublishNotificationTest.java

```diff
@@ -0,0 +1,268 @@
+        when(collection.getDescription()).thenReturn(cdmock);
+        when(collection.getDescription().getId()).thenReturn("cake-1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef");
+        when(collection.getDescription().getPublishDate()).thenReturn(clearCacheDateTime);
+    }
+
+    @Test
+    public void payloadCacheAPIIsCorrect() {
+        try (MockedStatic<Configuration> mocked = mockStatic(Configuration.class)) {
+            Configuration mockConfiguration = mock(Configuration.class);
+            when(mockConfiguration.isCacheAPIEnabled()).thenReturn(true);
+
+            ArrayList<String> urls = new ArrayList<>();
+            urls.add("/economy/...");
+            PublishNotification publishNotification = new PublishNotification(collection, urls, null);
+            assertNotNull(publishNotification);
+
+            PublishNotification.NotificationPayloadCacheApi payload = publishNotification.getPayloads().get(0);
+            String payloadJson = Serialiser.serialise(payload);
+            String payloadExpected = "{" +
+                    "\"collection_id\":\"cake-1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef\"," +
+                    "\"release_time\":\"2021-01-05T17:00:30.000Z\"," +
+                    "\"path\":\"/economy\"" +
+                    "}";
+            assertEquals(payloadExpected, payloadJson);
+            publishNotification = null;
+        } catch (Exception e) {
+
+        }
+    }
+
+    @Test
+    public void canCreatePublishNotificationWithURLsSetToNullWhenCacheEnabled() {
+        try (MockedStatic<Configuration> mocked = mockStatic(Configuration.class)) {
+            Configuration mockConfiguration = mock(Configuration.class);
+            when(mockConfiguration.isCacheAPIEnabled()).thenReturn(true);
+
+            PublishNotification publishNotification = new PublishNotification(collection, null, null);
+            assertNotNull(publishNotification);
+        }
+
+    }
+
+    @Test
+    public void canCreatePublishNotificationWithURLsSetToNullWhenCacheDisabled() {
+        try (MockedStatic<Configuration> mocked = mockStatic(Configuration.class)) {
+            Configuration mockConfiguration = mock(Configuration.class);
+            when(mockConfiguration.isCacheAPIEnabled()).thenReturn(false);
+
+            PublishNotification publishNotification = new PublishNotification(collection, null, null);
+            assertNotNull(publishNotification);
+        }
+
+    }
+
+    @Test
+    public void publishNotificationWhenCacheAPIEnabledTest() {
+        try (MockedStatic<Configuration> mocked = mockStatic(Configuration.class)) {
+
+            // Create a mock for the static method to return
+            Configuration mockConfiguration = mock(Configuration.class);
+            when(mockConfiguration.isCacheAPIEnabled()).thenReturn(true);
+
+            ArrayList<Host> hosts = new ArrayList();
```

Comment:
```suggestion
            ArrayList<Host> hosts = new ArrayList<>();
```

I can spot a lot of errors in this file (and others in this PR) that should have been caught by your IDE. If you haven't already, please download IntelliJ IDEA (even if it's just the Community Edition), and go through all the changes you've made in this PR and make sure that no warnings are displayed. You don't need to fix any lines that you haven't modified, but anything new or changed should be in a good state, with no warnings.